### PR TITLE
first pass at function to get event service item

### PR DESCRIPTION
**Pull request by jPurush**
Refactored `getEventServiceUrl` to add a new function `getEventServiceItems` in order for [hub-indexer](https://github.com/ArcGIS/hub-indexer/blob/9677cf6f94d61cc941bb86425963998f27e13f08/packages/api/middleware/events-fetch-feature-service.js#L18) to be able to get the public feature service url.

### CODE REVIEW

File: packages/events/src/util.ts [hunk 1]
```diff
@@ -21,60 +22,56 @@ import { getHubUrl } from "@esri/hub-domains";
  * Fetch the events service associated with a Hub Site.
  * @param orgId - Identifier of the ArcGIS Online Organization
```
Comment (jgravois) [inline packages/events/src/util.ts]: to differentiate between these new methods, i think our best option is to be explicit in the TSDoc code snippet about what is returned

```
* request("http://custom.maps.arcgis.com/sharing/rest/portals/self")
*   .then(response => {
*     getEventServiceUrl(response.id)
*       .then(url)
*         //  "https://hub.arcgis.com/..." or "https://services.arcgis.com/..."
*   })
```

File: packages/events/src/util.ts [hunk 2]
```diff
@@ -21,60 +22,56 @@ import { getHubUrl } from "@esri/hub-domains";
...
 
+/**
+ * ```js
+ * import { request } from "@esri/arcgis-rest-request";
+ * import { getEventServiceUrl } from "@esri/hub-events";
+ * // org ids can be retrieved via a call to portals/self
+ * request("http://custom.maps.arcgis.com/sharing/rest/portals/self")
+ *   .then(response => {
+ *     getEventServiceUrl(response.id)
+ *       .then(url)
```
Comment (jgravois) [inline packages/events/src/util.ts]: this snippet needs updated.

File: packages/events/src/util.ts [hunk 3]
```diff
@@ -118,3 +115,60 @@ export function getEventQueryFromType(
...
 }
+
+/**
+ * ```js
+ * import { request } from "@esri/arcgis-rest-request";
+ * import { getEventServiceItem } from "@esri/hub-events";
+ * // org ids can be retrieved via a call to portals/self
+ * request("http://custom.maps.arcgis.com/sharing/rest/portals/self")
+ *   .then(response => {
+ *     getEventServiceItems(response.id)
```
Comment (jgravois) [inline packages/events/src/util.ts]: ~~`getEventServiceItems`~~ > `getEventServiceItem`

File: packages/events/src/util.ts [hunk 4]
```diff
@@ -21,60 +22,56 @@ import { getHubUrl } from "@esri/hub-domains";
  * Fetch the events service associated with a Hub Site.
```
Comment (tomwayson) [inline packages/events/src/util.ts]: "Get the Hub REST API endpoint to use for an organization's events"

**Pull request merged**